tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -153,113 +153,113 @@
+        <item>Don\'t use</item>
     </string-array>
     <string-array name="pref_common_use_busybox_values">
         <item>1</item>
         <item>2</item>
         <item>3</item>
         <item>4</item>
     </string-array>
-    <string name="pref_common_categ_hotspot">HOTSPOT-Experimental</string>
-    <string name="pref_common_tethering_autostart">Start Tethering On Boot</string>
+    <string name="pref_common_categ_hotspot">Experimental HOTSPOT Settings</string>
+    <string name="pref_common_tethering_autostart">Start Tethering with Device</string>
     <string name="pref_common_tor_tethering">Allow Tor Tethering</string>
-    <string name="pref_common_tor_route_all">Route All through Tor</string>
-    <string name="pref_common_tor_route_all_summ">Route All incoming traffic through Tor</string>
-    <string name="pref_common_unblock_with_tor">Select WebSites</string>
-    <string name="pref_common_unblock_with_tor_summ">WebSites list to open with Tor for HOTSPOT</string>
-    <string name="pref_common_excludes_from_tor">Exclude WebSites</string>
-    <string name="pref_common_excludes_from_tor_summ">WebSites list to open directly for HOTSPOT</string>
+    <string name="pref_common_tor_route_all">All through Tor</string>
+    <string name="pref_common_tor_route_all_summ">Route all incoming traffic through Tor.</string>
+    <string name="pref_common_unblock_with_tor">Select Websites</string>
+    <string name="pref_common_unblock_with_tor_summ">List of websites to open with Tor for HOTSPOT.</string>
+    <string name="pref_common_excludes_from_tor">Excluded Websites</string>
+    <string name="pref_common_excludes_from_tor_summ">List of websites to open directly for HOTSPOT.</string>
     <string name="pref_common_itpd_tethering">Allow I2P Tethering</string>
     <string name="pref_common_fix_ttl">Fix TTL</string>
     <string name="pref_common_fix_ttl_summ">TTL will be fixed to 64 using a local VPN. Kernel support is not required.
-        It can only be used in \"root mode\" with the \"Run Modules With Root\" option disabled.
-        To fix TTL please start DNSCrypt or/and Tor.</string>
-    <string name="pref_common_local_eth_device_addr">LAN device IP address</string>
-    <string name="pref_common_local_eth_device_addr_summ">Enable Fix TTL and configure LAN device to connect to InviZible. Use default gateway: %s, DNS server: %s.</string>
-    <string name="pref_common_tethering_settings">Open Tethering config</string>
+        It can only be used in \"Root Mode\" with the \"Run Modules With Root\" off.
+        To fix TTL, please start DNSCrypt or/and Tor.</string>
+    <string name="pref_common_local_eth_device_addr">IP Address of LAN Device</string>
+    <string name="pref_common_local_eth_device_addr_summ">Enable \"Fix TTL\" and set up a LAN device to connect to InviZible. Use the default gateway: %s, DNS server: %s.</string>
+    <string name="pref_common_tethering_settings">Open the Tethering Setup</string>
```

Comment:
This is a good example of why actionable items could have a trailing ellipsis (…)
If not, it could possibly be "An open tethering setup" in translation/understanding.

Reply:
Just don't use trailing ellipsis. They only look good in short notifications or pop-up messages.

Reply:
Yes, I put in "the" to avoid that here.